### PR TITLE
added more options for attachment export

### PR DESCRIPTION
**Pull request by maavuz**

### CODE REVIEW

**Comment by thehelvetian** [comment]
Thank you @maavuz for your contribution. I'm going to comment a few things in the code, hopefully you can adjust.

File: wp-attachment-export.php [hunk 1]
```diff
@@ -58,6 +91,23 @@ function admin_screen() {
...
 			<form action="" method="get" id="export-filters">
 				<p><label><input type="radio" name="content" value="attachment" checked="checked" /> <?php esc_attr_e( 'Attachments', 'wp-attachment-export' ); ?></label></p>
+				<p><span class="label-responsive"><?php esc_attr_e( 'Authors', 'wp-attachment-export' ); ?></span>
+					<?php
+						global $wpdb;
+						$authors = $wpdb->get_col( "SELECT DISTINCT post_author FROM {$wpdb->posts} WHERE post_type = 'post'" );
+						wp_dropdown_users( array( 'include' => $authors, 'name' => 'post_author', 'multi' => true, 'show_option_all' => __('All') ) );
+					?>
+				</p>
+				<label for="attachment-start-date" class="label-responsive"><?php _e( 'Start date:' ); ?></label>
```
Comment (thehelvetian) [inline wp-attachment-export.php]: Please add an opening `<p>` tag before `<label>`

File: wp-attachment-export.php [hunk 2]
```diff
@@ -58,6 +91,23 @@ function admin_screen() {
...
+				<select name="attachment_start_date" id="attachment-start-date">
+					<option value="0"><?php _e( '&mdash; Select &mdash;' ); ?></option>
+					<?php hlvtn_WP_Attachment_Export::create_export_date_options( 'attachment' ); ?>
+				</select>
+				<label for="attachment-end-date" class="label-responsive"><?php _e( 'End date:' ); ?></label>
+				<select name="attachment_end_date" id="attachment-end-date">
+					<option value="0"><?php _e( '&mdash; Select &mdash;' ); ?></option>
+					<?php hlvtn_WP_Attachment_Export::create_export_date_options( 'attachment' ); ?>
```
Comment (thehelvetian) [inline wp-attachment-export.php]: Please use `self::create_export_date_options(...`

File: wp-attachment-export.php [hunk 3]
```diff
@@ -58,6 +91,23 @@ function admin_screen() {
...
+					<?php
+						global $wpdb;
+						$authors = $wpdb->get_col( "SELECT DISTINCT post_author FROM {$wpdb->posts} WHERE post_type = 'post'" );
+						wp_dropdown_users( array( 'include' => $authors, 'name' => 'post_author', 'multi' => true, 'show_option_all' => __('All') ) );
+					?>
+				</p>
+				<label for="attachment-start-date" class="label-responsive"><?php _e( 'Start date:' ); ?></label>
+				<select name="attachment_start_date" id="attachment-start-date">
+					<option value="0"><?php _e( '&mdash; Select &mdash;' ); ?></option>
+					<?php hlvtn_WP_Attachment_Export::create_export_date_options( 'attachment' ); ?>
```
Comment (thehelvetian) [inline wp-attachment-export.php]: Please use `self::create_export_date_options(...`

File: wp-attachment-export.php [hunk 4]
```diff
@@ -58,6 +91,23 @@ function admin_screen() {
...
+				</select>
+				<label for="attachment-end-date" class="label-responsive"><?php _e( 'End date:' ); ?></label>
+				<select name="attachment_end_date" id="attachment-end-date">
+					<option value="0"><?php _e( '&mdash; Select &mdash;' ); ?></option>
+					<?php hlvtn_WP_Attachment_Export::create_export_date_options( 'attachment' ); ?>
+				</select>
```
Comment (thehelvetian) [inline wp-attachment-export.php]: Please add a closing `</p>` tag after `</select>`

**Comment by maavuz** [comment]
Hi @thehelvetian. thank you pointing out those issue :) i have just fixed them you might wanna take a look in to that :)

**Comment by thehelvetian** [comment]
Thanks a lot for your contribution. I have merged your request and published a new version.

**Pull request merged**